pyop2/configuration.py

```diff
@@ -82,28 +77,23 @@ class Configuration(dict):
     """
     # name, env variable, type, default, write once
     DEFAULTS = {
-        "compiler": ("PYOP2_BACKEND_COMPILER", str, "gnu"),
-        "simd_isa": ("PYOP2_SIMD_ISA", str, "sse"),
+        "compiler": ("PYOP2_BACKEND_COMPILER", str, "gcc"),
+        "simd_width": ("PYOP2_SIMD_WIDTH", int, 4),
```

Comment:
I would prefer this to be automatically correctly determined based on what the chip supports. One does this by prodding the `cpuid` instruction at the processor (at least for x86). There are various python packages that allow this (e.g. https://github.com/flababah/cpuid.py/blob/master/cpuid.py). Does anyone else (@dham / @inducer) have cleverer ideas?

Reply:
That's definitely the plan, though currently these settings are not used.